tools/topology/topology1/sof/pipe-volume-playback.m4

```diff
@@ -80,7 +80,8 @@ indir(`define', concat(`PIPELINE_PCM_', PIPELINE_ID), Passthrough Playback PCM_I
 # PCM Configuration
 
 #
-PCM_CAPABILITIES(Passthrough Playback PCM_ID, CAPABILITY_FORMAT_NAME(PIPELINE_FORMAT), PCM_MIN_RATE, PCM_MAX_RATE, 2, PIPELINE_CHANNELS, 2, 16, 192, 16384, 65536, 65536)
+PCM_CAPABILITIES(Passthrough Playback PCM_ID, CAPABILITY_FORMAT_NAME(PIPELINE_FORMAT),
+			PCM_MIN_RATE, PCM_MAX_RATE, PIPELINE_CHANNELS, PIPELINE_CHANNELS, 2, 16, 192, 16384, 65536, 65536)
```

Comment:
This is to support playing stereo on a 4 channel pipeline.

Reply:
If compared with pipe-volume-capture.m4, you can find it also fixed min = max, and this, for now, should be the same with hardware, so I think it should be min channel = max channel. And here is a definition, you can set channel is 4 in your topology file.

Reply:
@TangleZ please remove this commit from the current PR, and will send a separate patch.

Reply:
Removed.

Reply:
@lgirdwood Removed this change, will find a better way to address this.